apps/museums/fields/index.js

```diff
@@ -23,5 +23,12 @@ module.exports = {
       className: 'visuallyhidden'
     },
     options: ['yes', 'no']
+  },
+  'contact-address': {
+    mixin: 'radio-group',
+    validate: 'required',
+    legend: {
+      className: 'visuallyhidden'
+    }
   }
 };
```

Comment:
ignore this change it's from lenny's PR https://github.com/UKHomeOffice/firearms/pull/113